nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core.Test/Utilities/MSBuildHelperTests.cs

```diff
@@ -167,7 +167,10 @@ public async Task AllPackageDependenciesCanBeTraversed()
             new("System.Threading.Tasks.Extensions", "4.5.4", DependencyType.Unknown),
             new("NETStandard.Library", "2.0.3", DependencyType.Unknown),
         };
-        var actualDependencies = await MSBuildHelper.GetAllPackageDependenciesAsync(temp.DirectoryPath, temp.DirectoryPath, "netstandard2.0", new[] { new Dependency("Microsoft.Extensions.Http", "7.0.0", DependencyType.Unknown) });
+        var actualDependencies = await MSBuildHelper.GetAllPackageDependenciesAsync(
```

Comment:
Any long lines containing argument/parameters lists that were hard wrapped should have all elements wrapped and indented with closing paren on the same line as the last element